src/scss/grommet-core/_objects.video.scss

```diff
@@ -37,6 +37,10 @@
 
   video {
     width: 100%;
+
+    .video--poster {
+      width: 100vw;
```

Comment:
I wouldn't think the poster would always be 100vw. What if the video is a smaller size?